src/actions/Packages/packageActions.js

```diff
@@ -7,15 +7,12 @@ const api = Api.getInstance();
 
 export const getPackages = () => {
 	return (dispatch) => {
-		dispatch(setIsQuerying(true));
```

Comment:
Why are we getting rid of isQuerying?

Reply:
setIsQuerying() was a method I wrote before to check if dependent/remote data is loading.  I didn't like the solution at the time, since I'm essentially putting API state into the data model that stores API results.  I was able to remove the isQuerying state (and its reducer actions) in favor of isRemoteDataLoaded() in the component.  It keeps the solution closer to the need.